### PR TITLE
[BXMSPROD-901] - changed bot name

### PR DESCRIPTION
**Pull request by Kaitou786**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "github-ci-bot",
+  "name": "kie-ci-bot",
```
Comment (Ginxo) [inline package.json:2]: @Kaitou786 I'm missing github somehow, it looks like a very generic bot here. What about `kie-github-ci-bot`?

Comment (Kaitou786) [reply package.json:2]: Let me see, if we can take this.

Comment (Kaitou786) [reply package.json:2]: other option would be `kie-github-bot`

Comment (Kaitou786) [reply package.json:2]: Both are available, wdyt `kie-github-bot` or `kie-github-ci-bot` ?

Comment (Ginxo) [reply package.json:2]: I think `kie-github-ci-bot` is more precise, ok?

Comment (Kaitou786) [reply package.json:2]: Ok, updating.

**Pull request merged**